e2e-tests/cypress/support/appeal-submission-appeal-site-address/confirmSiteAddressWasAccepted.js

```diff
@@ -1,6 +1,6 @@
 module.exports = () => {
   // confirm we are in the right place
-  cy.url().should('include', '/appellant-submission/site-ownership');
+  cy.url().should('include', '/appeal-householder-decision/site-ownership');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks